components/automate-ui/src/app/pages/project/rules/project-rules.component.ts

```diff
@@ -87,13 +86,6 @@ export class ProjectRulesComponent implements OnInit, OnDestroy {
     }
 
   ngOnInit(): void {
-    this.conflictErrorEvent.pipe(takeUntil(this.isDestroyed))
-    .subscribe((isConflict: boolean) => {
-      this.conflictError = isConflict;
-      // Open the ID input on conflict so user can resolve it.
-      this.modifyID = true;
-    });
```

Comment:
PR Bonus: No need for an event-emitter in this file, as the output is only needed in this file, so the actions here were inlined.